syncstorage/storage/__init__.py

```diff
@@ -47,6 +47,12 @@ class InvalidOffsetError(StorageError, ValueError):
     pass
 
 
+class InvalidBatch(StorageError, ValueError):
```

Comment:
naming nit: this should be with `InvalidBatchError` for consistency with its sibling class names.